models/scripts/PCA.py

```diff
@@ -0,0 +1,66 @@
+from tabpy_tools.client import Client
+import pandas as pd
+from numpy import array
+from sklearn.decomposition import PCA as sklearnPCA
+from sklearn.preprocessing import StandardScaler
+from sklearn.preprocessing import LabelEncoder
+from sklearn.preprocessing import OneHotEncoder
+import sys
+
+
+def PCA(component, _arg1, _arg2, *_argN):
+    '''
+    Principal Component Analysis is a technique that extracts the key
+    distinct components from a high dimensional space whie attempting
+    to capture as much of the variance as possible. For more information
+    on the function and how to use it please refer to tabpy-tools.md
+    '''
+    cols = [_arg1, _arg2] + list(_argN)
+    encodedCols = []
+    labelEncoder = LabelEncoder()
+    oneHotEncoder = OneHotEncoder(categories='auto', sparse=False)
+
+    for col in cols:
+        if (type(col[0]) is int or type(col[0]) is float):
```

Comment:
More idiomatic to write this as `isinstance(col[0], (int, float))`

Reply:
fixed